src/pages/ShareCodePage.tsx

```diff
@@ -23,17 +24,30 @@ import * as Url from '@libs/Url';
 import * as UserUtils from '@libs/UserUtils';
 import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
-import type {Report} from '@src/types/onyx';
+import type {Policy, Report} from '@src/types/onyx';
 
 type ShareCodePageOnyxProps = {
     /** The report currently being looked at */
     report?: OnyxEntry<Report>;
+
+    /** The policy for the report currently being looked at */
+    policy?: OnyxEntry<Policy>;
 };
 
 type ShareCodePageProps = ShareCodePageOnyxProps;
 
-function ShareCodePage({report}: ShareCodePageProps) {
+// In case of sharing a policy (workspace) only return user defined avatar. Default ws avatars have separate logic
+function getLogoForWorkspace(policy?: OnyxEntry<Policy>) {
```

Comment:
Add a return type here (the rule is to do it for all types except void, also for react components we don't define return types)